gradle-plugin/src/main/java/com/contrastsecurity/gradle/plugin/ContrastGradlePlugin.java

```diff
@@ -8,11 +8,18 @@
  * href=https://contrast.atlassian.net/browse/JAVA-8252>JAVA-8252</a>
  */
 public class ContrastGradlePlugin implements Plugin<Project> {
+
   public void apply(final Project target) {
+
+    ContrastConfigurationExtension extension =
+        target.getExtensions().create(EXTENSION_NAME, ContrastConfigurationExtension.class);
+
     target
         .getTasks()
         .register("hello", task -> task.doLast(s -> System.out.println("HelloWorld!")));
 
-    target.getTasks().register("empty", EmptyTask.class);
+    target.getTasks().register("installAgent", InstallAgentTask.class);
```

Comment:
This does not yet auto apply to test tasks, there's a bunch more configuration that needs to be decided for that. That is slated for https://contrast.atlassian.net/browse/JAVA-8882